src/wire/ipv6.rs

```diff
@@ -0,0 +1,372 @@
+use core::fmt;
+
+use byteorder::{ByteOrder, NetworkEndian};
+
+pub use super::IpProtocol as Protocol;
+
+/// A sixteen-octet IPv6 address.
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Copy, Default)]
+pub struct Address(pub [u8; 16]);
+
+impl Address {
+    /// An unspecified address.
+    pub const UNSPECIFIED: Address = Address([0x00; 16]);
+
+    /// Link local all routers multicast address
+    pub const LINK_LOCAL_ALL_NODES: Address = Address([0xff, 0x02, 0x00, 0x0,
```

Comment:
These look ugly, perhaps add a line break after `=`, indent, and then lay out in two lines?